### ISSUE TITLE
Is this code is updated?

### ISSUE THREAD

**Issue by sheikhrehman** [jonseg/crud-admin-generator#126]

**Comment by hamzaouiii** [comment]
No. it has been 3 years since the last commit on this project. It is basically abandoned.

**Comment by Muhammadinaam** [comment]
This is a nice project but I think developer has stopped working on it. Please try this new project: https://github.com/Muhammadinaam/speed-admin
